lib/client.go

```diff
@@ -33,13 +23,23 @@ import (
 	idemixcred "github.com/hyperledger/fabric-ca/lib/client/credential/idemix"
 	x509cred "github.com/hyperledger/fabric-ca/lib/client/credential/x509"
 	cidemix "github.com/hyperledger/fabric-ca/lib/common/idemix"
+	idemix2 "github.com/hyperledger/fabric-ca/lib/server/idemix"
```

Comment:
Might be good to have more expressive names than `idemix` and `idemix2`.

Reply:
`sidemix` should do the job